### PR TITLE
Changed American Spelling to British

### PR DESCRIPTION
**Pull request by mumerfarooq000**
Description:
Changed the words Enrollment, Enroll, and Honor

ticket link: https://edlyio.atlassian.net/browse/LUMS-247

### CODE REVIEW

**Review by emZubair — APPROVED** [review]
LGTM
All these changes are the core changes, will face issue while updating to the next release.

File: cms/envs/common.py [hunk 1]
```diff
@@ -487,7 +487,7 @@
 
     # Avoid issue with https://blog.heroku.com/chrome-changes-samesite-cookie
     # Override was found here https://github.com/django/django/pull/11894
-    'django_cookies_samesite.middleware.CookiesSameSite',
+    # 'django_cookies_samesite.middleware.CookiesSameSite',
```
Comment (tasawernawaz) [inline cms/envs/common.py]: why this change was required?

File: lms/envs/common.py [hunk 2]
```diff
@@ -1246,7 +1246,7 @@ def _make_locale_paths(settings):
 
     # Avoid issue with https://blog.heroku.com/chrome-changes-samesite-cookie
     # Override was found here https://github.com/django/django/pull/11894
-    'django_cookies_samesite.middleware.CookiesSameSite',
+    # 'django_cookies_samesite.middleware.CookiesSameSite',
```
Comment (tasawernawaz) [inline lms/envs/common.py]: same here

**Review by tasawernawaz — CHANGES_REQUESTED** [review]
@mumerfarooq000 please revert all changes in `/i18n/*` and `locale`. We don't do changes in language files directly. The translation process is totally different and these files are autogenerated with commands.

**Comment by tasawernawaz** [comment]
also please add a link to the ticket in the description of the PR.

**Review by sheralim012 — APPROVED** [review]

**Pull request merged**